jcl/src/openj9.traceformat/share/classes/com/ibm/jvm/traceformat/TraceFormat.java

```diff
@@ -383,9 +386,9 @@ public static void main(String[] args) throws Exception {
 
 		context.message(context, "Completed processing of " + context.getTotalTracePoints() + " tracepoints with " + context.getWarningCount() + " warnings and " + context.getErrorCount() + " errors");
 		if (verbose.booleanValue()) {
-			end = System.currentTimeMillis();
-			float Mbps = (float) (recordsInData * context.getRecordSize()) / (float) (((float) (end - start) / 1000) * (1024 * 1024));
-			context.message(context, "Total processing time " + (end - start) + "ms (" + Mbps + "Mb/s)");
+			end = System.nanoTime();
+			float Mbps = (float) (recordsInData * context.getRecordSize()) / (float) (TimeUnit.NANOSECONDS.toSeconds(end - start) * (1024 * 1024));
+			context.message(context, "Total processing time " + (end - start) + "ns (" + Mbps + "Mb/s)");
```

Comment:
Please don't change the units reported to users:
```
	float Mbps = (recordsInData * context.getRecordSize()) / (1024.0f * 1024.0f * TimeUnit.NANOSECONDS.toSeconds(end - start));
	context.message(context, "Total processing time " + TimeUnit.NANOSECONDS.toMillis(end - start) + "ms (" + Mbps + "Mb/s)");
```

Reply:
Updated.